### PR TITLE
Terraform code has beeen added

### PR DESCRIPTION
**Pull request by beginner-academy**
Multiple  files which creates aks cluster

### CODE REVIEW

File: storage_accounts.tf [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  private_dns               = local.combined_objects_private_dns
+  private_endpoints         = try(each.value.private_endpoints, {})
+  recovery_vaults           = local.combined_objects_recovery_vaults
+  storage_account           = each.value
+  var_folder_path           = var.var_folder_path
+  vnets                     = local.combined_objects_networking
+  virtual_subnets           = local.combined_objects_virtual_subnets
+
+  base_tags           = local.global_settings.inherit_tags
+  resource_group      = local.combined_objects_resource_groups[try(each.value.resource_group.lz_key, local.client_config.landingzone_key)][try(each.value.resource_group_key, each.value.resource_group.key)]
```
Comment (mallikarjunasiddappa) [inline storage_accounts.tf:24]: Code need to modify as expected

**Review by mallikarjunasiddappa — APPROVED** [review]
As discussed I am approving based on your explanation

**Review by shadowninja1990 — APPROVED** [review]
Looks good. Approved.

File: storage_accounts.tf (hunk 1 above)
Comment (834431csd) [reply storage_accounts.tf:24]: Approved

**Review by 834431csd — APPROVED** [review]
Approved

File: storage_accounts.tf [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  for_each = var.storage_accounts
+
+  client_config             = local.client_config
+  diagnostic_profiles       = try(each.value.diagnostic_profiles, {})
+  diagnostic_profiles_blob  = try(each.value.diagnostic_profiles_blob, {})
+  diagnostic_profiles_queue = try(each.value.diagnostic_profiles_queue, {})
+  diagnostic_profiles_table = try(each.value.diagnostic_profiles_table, {})
+  diagnostic_profiles_file  = try(each.value.diagnostic_profiles_file, {})
+  diagnostics               = local.combined_diagnostics
+  global_settings           = local.global_settings
```
Comment (kokkalakiuk) [inline storage_accounts.tf:13]: All good, hence approving this request

**Review by kokkalakiuk — APPROVED** [review]

**Review by Jag-Amg — APPROVED** [review]

**Review by shilpakotha12test — APPROVED** [review]
Approved

**Pull request merged**